app/frontend/stores/jurisdiction-store.ts

```diff
@@ -1,39 +1,62 @@
 import { Instance, flow, types } from "mobx-state-tree"
+import * as R from "ramda"
+import { withEnvironment } from "../lib/with-environment"
+import { withRootStore } from "../lib/with-root-store"
 import { IJurisdiction, JurisdictionModel } from "../models/jurisdiction"
 
 export const JurisdictionStoreModel = types
-  .model("JurisdictionStore", {
-    jurisdictionsMap: types.map(JurisdictionModel),
+  .model("JurisdictionStore")
+  .props({
+    jurisdictionMap: types.map(JurisdictionModel),
   })
+  .extend(withEnvironment())
+  .extend(withRootStore())
   .views((self) => ({
     // View to get a Jurisdiction by id
     getJurisdictionById(id: string) {
-      return self.jurisdictionsMap.get(id)
+      return self.jurisdictionMap.get(id)
     },
 
     // View to get all jurisdictions as an array
-    get Jurisdictions() {
-      return Array.from(self.jurisdictionsMap.values())
+    get jurisdictions() {
+      return Array.from(self.jurisdictionMap.values())
```

Comment:
I think values already returns an array. Array.from should not be needed

Reply:
This is the correct way to avoid typescript type errors